core/src/main/java/io/aiven/klaw/service/AclControllerService.java

```diff
@@ -111,6 +119,13 @@ public ApiResponse createAcl(AclRequestsModel aclRequestsModel) throws KlawExcep
       }
     }
 
+    if (kafkaFlavor.equals(KafkaFlavors.AIVEN_FOR_APACHE_KAFKA.value)) {
+      if (verifyServiceAccountsOfTeam(aclRequestsModel, tenantId))
+        return ApiResponse.builder()
+            .result("Failure : Mentioned Service account used by another team.")
```

Comment:
I think it would be a good idea for us to start adding the HTTP Status code in the ApiResponse in the status field e.g. here as 403 or 400 so that the FE can start using that as the error they return to the UI.

Reply:
Agree, but we need to consolidate all these error messages .. may be in a different PR